app/models/evaluation.rb

```diff
@@ -85,12 +85,12 @@ def self.parse_filters(filters)
       when 'iso3'
         countries = options
         site_ids << countries.map{ |iso3| Country.find_by(iso3: iso3).sites.pluck(:id) }
-        where_params[:sites] = "site_id IN (#{site_ids.join(',')})" unless options.empty?
+        where_params[:sites] = options.empty? ? nil : "site_id IN (#{site_ids.join(',')})"
       when 'methodology'
         options = options.map{ |e| "'#{e}'" }
-        where_params[:methodology] = "#{filter["name"]} IN (#{options.join(',')})" unless options.empty?
+        where_params[:methodology] = options.empty? ? nil : where_params[:methodology] = "#{filter["name"]} IN (#{options.join(',')})"
```

Comment:
You can remove the assignment in the else statement as you are already assigning the resulting value of this conditional to `where_params[:methodology]`

Reply:
Done!